astroplan/scheduling.py

```diff
@@ -34,13 +80,11 @@ def __repr__(self):
             return orig_repr.replace('object at', s)
 
     @classmethod
-    def from_exposures(cls, target, timeperexp, nexp, readouttime=0*u.second,
-                            configuration={}):
-        duration = nexp*(timeperexp + readouttime)
-        ob = cls(target, duration, configuration)
-        ob.timeperexp = timeperexp
-        ob.nexp = nexp
-        ob.readouttime = readouttime
+    def from_imagesets(cls, target, imageset_list, priority=1.0):
+        duration = 0
+        for IS in imageset_list:
```

Comment:
This is nitpicky, but the traditional pep8 solution for situations where you can't use a keyword is to add an `_`.  I.e. ``is``->``is_``, not ``is``->``IS``.

(Not a big deal at all, but thought I would mention it)